api/web_components/index.md

```diff
@@ -0,0 +1,174 @@
+
+## Examples
+
+- [Static page with all chart types playground](https://lit.dev/playground/#gist=822ce6018bb41113c866d703760c1def) ([source](/assets/examples/web-components/all-charts.html)):
+- [Dynamically updating charts playground](https://lit.dev/playground/#gist=9e3ac88e162248f849dd276ff5895ad0) ([source](/assets/examples/web-components/dynamic-map.html))
+
+## Variables and places
+
+Data Commons web components visualize statistical variables about one or more places.
+Variables and places are identified by [Data Commons Identifiers](/glossary.html#dcid), or [DCID](/glossary.html#dcid)s.
+
+To find the [DCID](/glossary.html#dcid) of a place or variable:
+
+1. Browse all 175K+ variables with the [Data Commons Statistical Variable Explorer](https://datacommons.org/tools/statvar).
+
+2. Search for places and variables with the [Data Commons Search](https://datacommons.org/search) page.
+
+3. Use the [Data Commons Graph Browser](https://datacommons.org/browser) to
+   understand the relationship between entities.
+
+> Example #1: Inspecting [Health / Health Insurance (Household) / No Health Insurance / Households Without Health Insurance](https://datacommons.org/tools/statvar#sv=Count_Household_NoHealthInsurance) shows us that the statistical variable `Count_Household_NoHealthInsurance` is available in the `United States` ([DCID](/glossary.html#dcid): `country/USA`) at `State`, `County`, and `City` levels.
+
+<img src="/assets/images/web_components/stat-var-explorer.png" width="600"/>
+
+> Example #2 , the graph browser [country/USA](https://datacommons.org/browser/country/USA) page shows the [DCID](/glossary.html#dcid)s for all US states and territories.
+
+<img src="/assets/images/web_components/graph-explorer.png" width="600"/>
+
+## Styling
+
+Custom styles are supported through [CSS shadow parts](https://developer.mozilla.org/en-US/docs/Web/CSS/::part).
+
+| CSS `::part`                          | Description                                                                                                 | Components                 |
+| ------------------------------------- | ----------------------------------------------------------------------------------------------------------- | -------------------------- |
+| `container`                           | Chart container element                                                                                     | All                        |
+| `legend`                              | Chart legend                                                                                                | `bar`, `line`, `map`,`pie` |
+| `place-path`                          | Geo boundary                                                                                                | `map`                      |
+| `place-path-<dcid>`                   | Geo boundary for a particular place. Example: `place-path-geoId\/12` \*                                     | `map`                      |
```

Comment:
I think it would be worth specifically highlighting the need to escape the slashes, since that will be very common with our dcid's

Reply:
Moved the note about needing to escape slashes up, and put it in a red alert box to make the messaging more prominent.